ipn/ipnlocal/local.go

```diff
@@ -3740,11 +3740,15 @@ func updateExitNodeUsageWarning(p ipn.PrefsView, state *netmon.State, healthTrac
 }
 
 func (b *LocalBackend) checkExitNodePrefsLocked(p *ipn.Prefs) error {
-	if err := featureknob.CanUseExitNode(); err != nil {
-		return err
+	tryingToUseExitNode := p.ExitNodeIP.IsValid() || p.ExitNodeID != ""
+
```

Comment:
Nit - you could short-circuit here and return `nil` if `!tryingToUseExitNode`.

Reply:
Oh whoops good catch! Will adjust